### PR TITLE
feat(Wolvesville): add API details

### PR DESCRIPTION
**Pull request by BlackWither17**
## Acknowledgements
- [ ✅ ] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [ ✅ ] I linted the code by running `yarn format`
- [ ✅ ] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
image
image
image
image
image

<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/W/Wolvesville/presence.ts [hunk 1]
```diff
@@ -6,7 +6,10 @@ const presence = new Presence({
 const enum Assets {
 	WovHeroes = "https://cdn.rcd.gg/PreMiD/websites/W/Wolvesville/assets/1.png",
 	WovBlog = "https://cdn.rcd.gg/PreMiD/websites/W/Wolvesville/assets/2.png",
+	API = "https://i.imgur.com/d8PLSNR.png",
+	Legal = "https://imgur.com/pqasJnG.png",
 	Vouchers = "https://cdn.rcd.gg/PreMiD/websites/W/Wolvesville/assets/3.png",
+	Voting = "https://imgur.com/Nt9hAOP.png",
```
Comment (theusaf) [inline websites/W/Wolvesville/presence.ts:12]: Assets must be 512x512px

File: websites/W/Wolvesville/presence.ts [hunk 2]
```diff
@@ -70,8 +83,30 @@ presence.on("UpdateData", async () => {
...
+		} else if (document.location.href.includes("#/?id=")) {
+			if (!privacyMode) {
+				presenceData.details = "Reading the Public API docs:";
+				presenceData.state = findTitleByHref(
+					document.location.href.split("api-docs.wolvesville.com/")[1]
+				);
+				presenceData.buttons = [
+					{
+						label: "Read documentation",
+						url: document.URL,
```
Comment (theusaf) [inline websites/W/Wolvesville/presence.ts:102]:
```suggestion
						label: "Read Documentation",
						url: document.location.href,
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by theusaf** [comment]
Looks like the recent push just erased all changes previously made in this PR

**Comment by BlackWither17** [comment]
> Looks like the recent push just erased all changes previously made in this PR

Should I create a new merge request?

**Pull request merged**